scripts/pancancer.py

```diff
@@ -0,0 +1,341 @@
+#! /usr/bin/python3
+import json
+import configparser
+import time
+import sys
+import subprocess
+import logging
+from cliff.app import App
+from cliff.commandmanager import CommandManager
+from cliff.command import Command
+
+class WorkflowLister:
+    "Get a listing of workflows from a source of workflow metadata."
+    # Eventually, this data structure should come from some sort of online registry. For now, just define it in code.
+    _workflows= {   'HelloWorld':
```

Comment:
The workflow json content, doesn't belong in the script as you already have noted.